resteasy-dependencies-bom/pom.xml

```diff
@@ -79,6 +79,7 @@
         <version.org.wildfly.extras.creaper>1.6.1</version.org.wildfly.extras.creaper>
         <version.org.wildfly.wildfly-arquillian-container-managed>2.1.1.Final</version.org.wildfly.wildfly-arquillian-container-managed>
         <version.org.wildfly.wildfly-arquillian-container-remote>2.1.1.Final</version.org.wildfly.wildfly-arquillian-container-remote>
+        <version.org.wildfly.wildfly-microprofile-config-smallrye>14.0.0.Final</version.org.wildfly.wildfly-microprofile-config-smallrye>
```

Comment:
Any reason to this version vs. the latest one? https://mvnrepository.com/artifact/org.wildfly/wildfly-microprofile-config-smallrye